src/api/exchanges/include/upbitprivateapi.hpp

```diff
@@ -32,6 +34,17 @@ class UpbitPrivate : public ExchangePrivate {
   WithdrawInfo withdraw(MonetaryAmount, ExchangePrivate&) override { throw std::runtime_error(""); }
 
  private:
+  struct CurrenciesFunc {
```

Comment:
I liked the initial naming "tradable currencies".

Reply:
OK